.github/workflows/build.yml

```
@@ -0,0 +1,75 @@
+name: Build
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+env:
+  DEB_BUILD_DOCKER_IMAGE: "pitop/pi-top-os-deb-build"
+  DEB_BUILD_DOCKER_TAG: "latest"
+  CHANGELOG_AUTHOR_NAME: "pi-top"
+  CHANGELOG_AUTHOR_EMAIL: "deb-maintainers@pi-top.com"
+
+jobs:
+  build-debian-package:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2.2.0
+        with:
+          fetch-depth: 0
+
+      - name: Add changelog entry for latest snapshot version
+        uses: pi-top/git-debian-changelog-bump-action@master
+        with:
+          release: false
+          author_name: ${{ env.CHANGELOG_AUTHOR_NAME }}
+          author_email: ${{ env.CHANGELOG_AUTHOR_EMAIL }}
+
+      # We only want ARM builds. This step is here to provide the option of
+      # using a strategy matrix if additional builds (e.g. `arm64`) are desired
+      - name: Determine architecture to use from package info
+        # If architecture is 'all', then it can be compiled on host architecture
+        # All other pi-topOS cases require 'armhf'
+        run: |
+          target_architecture=amd64
+          if grep '^Architecture:' debian/control | grep -q -v 'all'; then
+            target_architecture=armhf
+          fi
+          echo "TARGET_ARCHITECTURE=${target_architecture}" >> $GITHUB_ENV
+
+      - name: Build Debian package
+        uses: pi-top/debian-package-build-action@crosscompiling
```

```suggestion
        uses: pi-top/debian-package-build-action@master
```
this should be changed to `master` once https://github.com/pi-top/debian-package-build-action/pull/11 is merged